.github/workflows/publish-to-test-pypi.yml

```diff
@@ -0,0 +1,74 @@
+name: Publish Python 🐍 distribution 📦 to TestPyPI
+
+on:
+  workflow_dispatch:  # Keep manual trigger
+    inputs:
+      rc_version:
+        description: 'RC version number (e.g., 1, 2, 3)'
+        required: true
+        type: string
+  schedule:
+    - cron: "0 0 * * *"  # Run every day at midnight
```

Comment:
if this needs an `rc_version` input how can it run automatically at night?

Reply:
Nope, if it is a nightly package, it will use the date as the rc_version.